folly/dynamic-inl.h

```diff
@@ -20,6 +20,7 @@
 #include <folly/CPortability.h>
 #include <folly/Conv.h>
 #include <folly/Format.h>
+#include <glog/logging.h>
```

Comment:
We should not add this include. I have an internal change here which I hope to ship soon.